phpcs/ruleset.xml

```diff
@@ -75,4 +75,16 @@
     <rule ref="Squiz.WhiteSpace.ScopeClosingBrace"/>
     <rule ref="Squiz.WhiteSpace.ScopeKeywordSpacing"/>
     <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace"/>
+    <rule ref="Squiz.Arrays.ArrayDeclaration">
+        <include name="Squiz.Arrays.ArrayDeclaration.NoCommaAfterLast" />
+        <include name="Squiz.Arrays.ArrayDeclaration.CommaAfterLast" />
+
+        <exclude name="Squiz.Arrays.ArrayDeclaration.ValueNoNewline" />
+        <exclude name="Squiz.Arrays.ArrayDeclaration.CloseBraceNotAligned" />
+        <exclude name="Squiz.Arrays.ArrayDeclaration.MultiLineNotAllowed" />
+        <exclude name="Squiz.Arrays.ArrayDeclaration.SingleLineNotAllowed" />
+        <exclude name="Squiz.Arrays.ArrayDeclaration.KeyNotAligned" />
+        <exclude name="Squiz.Arrays.ArrayDeclaration.ValueNotAligned" />
+        <exclude name="Squiz.Arrays.ArrayDeclaration.DoubleArrowNotAligned" />
```

Comment:
T'en pense quoi de l'ajouter au passage ? c'est déjà une convention que l'on s'est donné

```suggestion
        <include name="Squiz.Arrays.ArrayDeclaration.DoubleArrowNotAligned" />
```

Reply:
Je l'avais fait au début mais comme on en avait pas parlé je l'avais retiré :D so I'm all for it :)